### DISCUSSION TITLE
Import New Fonts

### DISCUSSION THREAD

**Discussion by Paratrasnet** [ScratchAddons/ScratchAddons#5427] [Ideas] [answered]
There are only a small number of fonts from which you can choose in scratch. My Idea is to make a button for importing fonts.

![image](https://user-images.githubusercontent.com/80710858/206497200-50780067-cce1-4f33-be80-3a729edaffde.png)

**Comment by BroJac5246** [comment, marked as answer]
See #2050.

**Reply by Paratrasnet** [reply]
Ok thank you.
